relnotes/array.feature.md

```diff
@@ -0,0 +1,26 @@
+### Assigns elements to a static or dynamic array.
+
+* `ocean.core.Array`
+
+  Add a new helper function `fill()` to assign all the elements of an array
+  array. The function is a replacement for declaring an array literal, it
+  does not allocate GC memory and ensures the length of the array matches the
```

Comment:
"and ensures the length of the array matches" -- does that imply that declaring an array literal _doesn't_ check that the lengths match?

Reply:
That is correct. Array literals don't require lengths to be identical. You can do `int[10] a = [1,2];`

Reply:
So I don't need to update the documentation here, right?